lib/containers/podman.pm

```diff
@@ -38,4 +40,46 @@ sub get_storage_driver {
     return $storage;
 }
 
+sub switch_network_backend {
+    my ($self, $net) = @_;
+    # exit if already cni present
+    record_info("backend", $net);
+    return 1
+      if (script_output("podman info --format {{.Host.NetworkBackend}}") =~ /^$net/);
+    my @pkgs;
+    if ($net =~ /^cni/) {
+        @pkgs = qw(cni cni-plugins);
+    } elsif ($net =~ /^netavark/) {
+        @pkgs = qw(netavark aardvark-dns);
+    } else {
+        return 0;
+    }
+    my $config = "/etc/containers/containers.conf";
+    # skip if already installed
+    unless (script_run("rpm -q @pkgs") eq 0) {
```

Comment:
We never check if packages are installed before installing.

Reply:
Eh in fact: if installed already, it is unuseful to run i.e. transactional update

Reply:
sure, but in that case you just need if `rpm` returns non-zero

hence `if (script_run("rpm -q @pkgs"))` or `if (script_run("rpm -q @pkgs") != 0)`